pkg/scoring/atlassian_test.go

```
@@ -0,0 +1,319 @@
+	}
+
+	m := atlassianMatch("info@example.com", "mycompany.atlassian.net")
+	fired, err := c.Evaluate(context.Background(), m)
+	assert.NoError(t, err)
+	assert.True(t, fired)
+}
+
+func TestAtlassianTokenExpiredCondition_NotExpired(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{"accountId":"123"}`))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)